docs/hooks/order-select.md

```diff
@@ -0,0 +1,366 @@
+# `order-select`
+
+| Metadata | Value
+| ---- | ----
+| specificationVersion | 1.0
+| hookVersion | 0.1.0
+
+## Workflow
+
+The `order-select` hook fires when a clinician selects one or more orders to place for a patient. 
+This hook is among the first workflow events for an order entering a draft status. 
+The context of this hook may include defaulted order details 
+as it occurs immediately upon the clinician selecting the order from the order catalogue of the CPOE, 
+but before she manually specifies order details (e.g. dose, quanity, route, etc). 
+Additionally, the context may include previously selected orders that are not yet signed from the same ordering session. 
+The `order-select` hook occurs after the clinician selects the order, but before she fills out (configures) the order. 
```

Comment:
Does it make sense to add a sentence on how this is different than the 'medication-prescribe' hook? 

'This is different than the 'medication-prescribe' which is triggered after a clinician completes the details or the order.'

Reply:
Per conversation, this definition of the workflow step at which the `medication-prescribe` hooks fires should live in a separate issue/PR specifically about `medication-prescribe`. Further, we should consider deprecating `medication-prescribe` in favor of a `medication-select` and `medication-sign`.